cmd/root.go

```diff
@@ -172,6 +172,16 @@ func rootCmd() *cobra.Command {
 	return rootCmd
 }
 
-func Execute() {
-	cobra.CheckErr(rootCmd().Execute())
+// func Execute() {
+// 	cobra.CheckErr(rootCmd().Execute())
+// }
+
+// control your exit status with Execute, by default in cmd/root.go:.
+func Execute() int {
+	if err := rootCmd().Execute(); err != nil {
+		// log it, then
+		return 1
+	}
+
+	return 0
```

Comment:
Doesn't main.go need updating?

There are several possible panics or `log.Fatal` calls in the subcommands. Should this also have a recover handler?

```suggestion
func Execute() (rc int) {
	defer func() {
		if r := recover(); r != nil {
			// handle the panic
			rc = 1
		}
	}()

	if err := rootCmd().Execute(); err != nil {
		rc = 1
	}

	return
```